lib/generators/shopify_app/shopify_app_generator.rb

```diff
@@ -9,6 +9,7 @@ def initialize(args, *options)
       def run_all_generators
         generate "shopify_app:install #{@opts.join(' ')}"
         generate "shopify_app:shop_model"
+        generate("shopify_app:authenticated_controller")
```

Comment:
I presume the parentheses are here because Rubocop wanted them. May as well add them to the other `generate` lines to match.